File: common-core/src/main/java/com/ityu/dao/system/DictRepository.java

package com.ityu.dao.system;



import com.ityu.bean.entity.system.Dict;
import com.ityu.dao.BaseRepository;

import java.util.List;

public interface DictRepository extends BaseRepository<Dict,Long> {
    List<Dict> findByPid(Long pid);
    List<Dict> findByNameAndPid(String name, Long pid);

    List<Dict> findByNameLike(String name);
}

File: common-core/src/main/java/com/ityu/common/dao/system/DictRepository.kt
package com.ityu.common.dao.system

import com.ityu.common.bean.entity.system.Dict
import com.ityu.common.dao.BaseRepository

interface DictRepository : BaseRepository<Dict?, Long?> {
    fun findByPid(pid: Long?): List<Dict?>?
    fun findByNameAndPid(name: String?, pid: Long?): List<Dict?>?
    fun findByNameLike(name: String?): List<Dict?>?
}
